fall18/p4/IMDB-Movie-Data.csv

```diff
@@ -0,0 +1,999 @@
+219,Kick-Ass,"Action,Comedy",Matthew Vaughn,"Aaron Taylor-Johnson, Nicolas Cage, Chloë Grace Moretz, Garrett M. Brown",2010,117,7.7,48.04
+220,Hardcore Henry,"Action,Adventure,Sci-Fi",Ilya Naishuller,"Sharlto Copley, Tim Roth, Haley Bennett, Danila Kozlovsky",2015,96,6.7,9.24
+221,Cars,"Animation,Adventure,Comedy",John Lasseter,"Owen Wilson, Bonnie Hunt, Paul Newman, Larry the Cable Guy",2006,117,7.1,244.05
+222,It Follows,"Horror,Mystery",David Robert Mitchell,"Maika Monroe, Keir Gilchrist, Olivia Luccardi,Lili Sepe",2014,100,6.9,14.67
+223,The Girl with the Dragon Tattoo,"Crime,Drama,Mystery",David Fincher,"Daniel Craig, Rooney Mara, Christopher Plummer,Stellan Skarsgård",2011,158,7.8,102.52M
+224,We're the Millers,"Comedy,Crime",Rawson Marshall Thurber,"Jason Sudeikis, Jennifer Aniston, Emma Roberts, Ed Helms",2013,110,7,150.37M
+225,American Honey,Drama,Andrea Arnold,"Sasha Lane, Shia LaBeouf, Riley Keough, McCaul Lombardi",2016,163,7,0.66
+226,The Lobster,"Comedy,Drama,Romance",Yorgos Lanthimos,"Colin Farrell, Rachel Weisz, Jessica Barden,Olivia Colman",2015,119,7.1,8.7
+227,Predators,"Action,Adventure,Sci-Fi",Nimród Antal,"Adrien Brody, Laurence Fishburne, Topher Grace,Alice Braga",2010,107,6.4,52
+228,Maleficent,"Action,Adventure,Family",Robert Stromberg,"Angelina Jolie, Elle Fanning, Sharlto Copley,Lesley Manville",2014,97,7,241.41
+229,Rupture,"Horror,Sci-Fi,Thriller",Steven Shainberg,"Noomi Rapace, Michael Chiklis, Kerry Bishé,Peter Stormare",2016,102,4.8,0
+230,Pan's Labyrinth,"Drama,Fantasy,War",Guillermo del Toro,"Ivana Baquero, Ariadna Gil, Sergi López,Maribel Verdú",2006,118,8.2,37.62
+231,A Kind of Murder,"Crime,Drama,Thriller",Andy Goddard,"Patrick Wilson, Jessica Biel, Haley Bennett, Vincent Kartheiser",2016,95,5.2,0
+232,Apocalypto,"Action,Adventure,Drama",Mel Gibson,"Gerardo Taracena, Raoul Max Trujillo, Dalia Hernández,Rudy Youngblood",2006,139,7.8,50.86
+233,Mission: Impossible - Rogue Nation,"Action,Adventure,Thriller",Christopher McQuarrie,"Tom Cruise, Rebecca Ferguson, Jeremy Renner, Simon Pegg",2015,131,7.4,195M
+234,The Huntsman: Winter's War,"Action,Adventure,Drama",Cedric Nicolas-Troyan,"Chris Hemsworth, Jessica Chastain, Charlize Theron, Emily Blunt",2016,114,6.1,47.95
+235,The Perks of Being a Wallflower,"Drama,Romance",Stephen Chbosky,"Logan Lerman, Emma Watson, Ezra Miller, Paul Rudd",2012,102,8,17.74
+236,Jackie,"Biography,Drama,History",Pablo Larraín,"Natalie Portman, Peter Sarsgaard, Greta Gerwig,Billy Crudup",2016,100,6.8,13.96
+237,The Disappointments Room,"Drama,Horror,Thriller",D.J. Caruso,"Kate Beckinsale, Mel Raido, Duncan Joiner, Lucas Till",2016,85,3.9,2.41
+238,The Grand Budapest Hotel,"Adventure,Comedy,Drama",Wes Anderson,"Ralph Fiennes, F. Murray Abraham, Mathieu Amalric,Adrien Brody",2014,99,8.1,59.07
+239,The Host,"Action,Adventure,Romance",Andrew Niccol,"Saoirse Ronan, Max Irons, Jake Abel, Diane Kruger",2013,125,5.9,26.62
```

Comment:
Change this to: 
239,The Host ,"Action,Adventure,Romance",Andrew Niccol,"Saoirse Ronan, Max Irons, Jake Abel, Diane Kruger",2013,125,5.9,26.62

Needed to add a space as there is another movie called The Host